### ISSUE TITLE
Unable to run ```SenseAct-NN Skill-Chain Baseline``` evaluation.

### ISSUE THREAD

**Issue by francescotaioli** [Ram81/goat-bench#24]
I'm unable to run the evaluation of the ```SenseAct-NN Skill-Chain Baseline``` (eval code at the end of the ```README```)

Content of  ```data/goat-assets/checkpoints/sense_act_nn_skill_chain``` is only ```language_nav_ckpt.pth```. See [here - HF](https://huggingface.co/datasets/axel81/goat-bench/tree/main/checkpoints/sense_act_nn_skill_chain).

When running the code, I got the following error:
```
File goat_bench/models/high_level_policy.py", line 124, in __init__
    self.ovon_policy = ovon_policy_cls.from_config(
AttributeError: 'NoneType' object has no attribute 'from_config'
```

Indeed, ```ovon_policy_cls``` should be 
```
ovon_policy_cls = baseline_registry.get_policy(
            "PointNavResnetCLIPPolicy"
        )
```

but ```PointNavResnetCLIPPolicy``` it is not registered. It seems to be refactored to ```GOATPolicy```, correct me if I'm wrong.
However, If I replace ```PointNavResnetCLIPPolicy```  with  ```GOATPolicy``` , several model keys seem to be missing. Indeed, I have the following error:
```
FileNotFoundError: [Errno 2] No such file or directory: 'data/new_checkpoints/ovon/ver/resnetclip_rgb_text/seed_1/ckpt.121.pth'
```

**Comment by rtabrizi** [comment]
same issue with me

**Comment by francescotaioli** [comment]
It seems also that ```language_nav_ckpt.pth``` is serialized with the old module name ```goat```

**Comment by rtabrizi** [comment]
@francescotaioli have you had any luck?

**Comment by francescotaioli** [comment]
@rtabrizi Nope, unfortunately, I could not run it, and in the meantime, I was focusing on other things. Hope they will release the new checkpoints

**Comment by rtabrizi** [comment]
@Ram81 are you able to upload the correct checkpoint?

**Comment by Ram81** [comment]
@rtabrizi @francescotaioli you are right. The checkpoints are outdated. I'll push the fixed version by the end of this week. Sorry about that

**Comment by rtabrizi** [comment]
Thank you!

**Comment by francescotaioli** [comment]
Hi @Ram81, sorry to bother you again. Do you have any updates on that?

**Comment by NormanMises** [comment]
want updates， too

**Comment by yyuncong** [comment]
Hi @Ram81, sorry to bother you with this. Could you provide any updates on this issue please?

**Comment by chashru** [comment]
any updates on this?
